src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = days * 40;
+
+  if (days >= 3 && days <= 6) {
+    totalAmount = totalAmount - 20;
+  } else if (days > 6) {
```

Comment:
if there is `else if`, then there should be `else` too. or you can just write several `if`